task2.js

```diff
@@ -0,0 +1,78 @@
+const Joi = require('@hapi/joi');
+
+const express = require('express');
+const collection = require('./db');
+const bodyQuerySchema = require('./validations');
+const bodyParser = require('body-parser');
```

Comment:
you don't have to use body parser since it is included in express by default. You can just call app.use(express.json())